esp-hal/src/dma/mod.rs

```diff
@@ -1877,6 +1911,7 @@ where
         &mut self,
         peri: DmaPeripheral,
         chain: &DescriptorChain,
+        enable_wrback: bool,
```

Comment:
I'm wondering if you can infer this from `DescriptorChain`, you should be able to see if it's circular based on the last descriptor.

Reply:
good idea 👍